### ISSUE TITLE
Admin cannot manage Roles  [ ] Devops box no permission

### ISSUE THREAD

**Issue by DaveT-T** [mastodon/mastodon#22467]
### Steps to reproduce the problem

Instance working well Sidekiq view window working well for months.

Used CLI to make second user Admin
RAILS_ENV=production DB_NAME=mastodon.greatlakes bin/tootctl accounts modify nico --role Admin
OK

Within moments, 
Sidekiq view window and menu option disappeared for all Admin users

Came across customizable permissions thread
[Add customizable user roles #18641]

Attempted to Make a new ROLE to assign [ ] DevOps permission returns error "cannot include permissions your current role does not possess"

Also no user can access the [ ] Administrator checkbox.

Admin Role has no permissions options to check or uncheck.

Somehow, Admin Role cannot view sidekiq or create new roles with all permissions.

is there a tootctl command to create new Role or modify Role permissions?

![manage roles 121822](https://user-images.githubusercontent.com/118393116/208323713-6d9f90fc-e43a-4678-925f-ad7d800a6a5e.jpg)

### Expected behaviour

Expected Admin user to have full permissions to set Roles

### Actual behaviour

Admin users do not have permission to set highest Role Permissions

### Specifications

SOFTWARE

Mastodon | 4.0.2
Ruby | 3.0.3p157
PostgreSQL | 15.1
Redis | 7.0.4

**Comment by ineffyble** [comment]
You want to create an Owner, not an Admin.

**Comment by DaveT-T** [comment]
Nevermind......   From documentation side I thought Owner was lowest default user perms.   I stumbled across roles.yml in live/config and that ecplained it all to me:

roles.yml shows
admin:
    position:  100  (I changed that in exploring issue)
    permissions:
      (long list)
    
owner:
    name: Owner
    position:  1000
    permissions:
    - administrator

As soon as I saw this file I realized that "Owner" is what I thought "Admin" was.

I changed user role to Owner and Sidekiq came back with full account and role control.

I will add a note at the other thread to help others....  Random how this occurred.
